commands/shooter/aimshootertotarget.py

```diff
@@ -19,53 +17,55 @@ def __init__(self, shooter: ShooterSubsystem, shooterOffset: Control2D) -> None:
+        # )
 
-        staticDifference = Transform2d(currentPose, constants.kSimDefaultTargetLocation)
-        staticRotation = rotationFromTranslation(staticDifference.translation())
+        # staticDifference = Transform2d(currentPose, constants.kSimDefaultTargetLocation)
+        # staticRotation = rotationFromTranslation(staticDifference.translation())
 
         ## NOTE: untested moving code, currently not in use
-        newPosition = Pose2d(
-            deltaPos.X() + currentPose.X(),
-            deltaPos.Y() + currentPose.Y(),
-            deltaPos.rotation() + currentPose.rotation(),
-        )
+        # newPosition = Pose2d(
+        #     deltaPos.X() + currentPose.X(),
+        #     deltaPos.Y() + currentPose.Y(),
+        #     deltaPos.rotation() + currentPose.rotation(),
+        # )
 
-        movingDifference = Transform2d(newPosition, constants.kSimDefaultTargetLocation)
-        movingRotation = rotationFromTranslation(movingDifference.translation())
+        # movingDifference = Transform2d(newPosition, constants.kSimDefaultTargetLocation)
+        # movingRotation = rotationFromTranslation(movingDifference.translation())
 
-        deltaRotation = movingRotation - staticRotation
+        # deltaRotation = movingRotation - staticRotation
 
-        if SmartDashboard.getBoolean(
-            constants.kTargetDistanceRelativeToRobotKeys.validKey, False
-        ):  # if we can obtain a distance, use that information
-            distance = (
-                SmartDashboard.getNumber(
-                    constants.kTargetDistanceRelativeToRobotKeys.valueKey, 0.0
-                )
-                + constants.kOffsetDistanceRange
-                * self.distanceAndRotationOffset.forwardsBackwards()
-            )
+        # if SmartDashboard.getBoolean(
+        #     constants.kTargetDistanceRelativeToRobotKeys.validKey, False
+        # ):  # if we can obtain a distance, use that information
+        #     distance = (
+        #         SmartDashboard.getNumber(
+        #             constants.kTargetDistanceRelativeToRobotKeys.valueKey, 0.0
+        #         )
+        #         + constants.kOffsetDistanceRange
+        #         * self.distanceAndRotationOffset.forwardsBackwards()
+        #     )
 
-            targetSpeed = constants.kShootingMappingFunction(distance)
-            self.shooter.setWheelSpeed(targetSpeed)
+        #     targetSpeed = constants.kShootingMappingFunction(distance)
+        #     self.shooter.setWheelSpeed(targetSpeed)
 
-            targetAngle = Rotation2d.fromDegrees(
-                constants.kHoodMappingFunction(distance)
-            )
-            self.shooter.setHoodAngle(targetAngle)
+        #     targetAngle = Rotation2d.fromDegrees(
+        #         constants.kHoodMappingFunction(distance)
+        #     )
+        #     self.shooter.setHoodAngle(targetAngle)
+        self.shooter.setHoodAngle(Rotation2d.fromDegrees(8))
+        self.shooter.setWheelSpeed(550)
```

Comment:
Already done in manual shooting

Reply:
the difference between this and manual is that it also includes the turret rotation, just with set speed and angle

we also do not want to remove the commented stuff as they are the calculations for wheel speed and hood angle which we plan to refine and use soon

Reply:
Commented code removed for now, still available in git history